packages/connect-widget/src/shared/localStorage/apple.ts

```diff
@@ -0,0 +1,20 @@
+import Constants from "../constants";
+import { isDemo } from "../api";
+
+const isAppleStorageKey = "is-apple";
+
+export const getIsApple = (searchParams: URLSearchParams): boolean | null => {
+  const isAppleParam = searchParams.get(Constants.APPLE_PARAM);
+
+  const appleParamTrue = isAppleParam && isAppleParam === "true";
+
+  if (appleParamTrue) {
+    if (!isDemo && localStorage) {
+      localStorage.setItem(isAppleStorageKey, isAppleParam);
+    }
+
+    return appleParamTrue;
+  }
+
+  return localStorage ? !!localStorage.getItem(isAppleStorageKey) : null;
```

Comment:
Why `null` if we can't determine whether is apple or not? Wouldn't it be better to return `false` since (it seems) we don't use `null` for anything in particular? Then this function can return pure `boolean`.